packages/build-info/src/frameworks/vite.ts

```diff
@@ -10,7 +10,7 @@ export class Vite extends BaseFramework implements Framework {
     '@remix-run/server-runtime',
     '@shopify/hydrogen',
     '@builder.io/qwik',
-    'solid-start',
+    '@solidjs/start',
     'solid-js',
```

Comment:
here too, let's keep the old name, and I think it could be useful to leave comments
```suggestion
    // Used this name up to 0.3.11
    'solid-start',
    // Renamed starting at 0.4.0
    '@solidjs/start',
    'solid-js',
```